pyproject.toml

```diff
@@ -1,67 +1,79 @@
-[tool.poetry]
+[build-system]
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
+build-backend = "poetry_dynamic_versioning.backend"
+
+[project]
 name = "linkml-runtime"
-version = "0.0.0"
 description = "Runtime environment for LinkML, the Linked open data modeling language"
 authors = [
-    "Chris Mungall <cjmungall@lbl.gov>",
-    "Harold Solbrig <solbrig@jhu.edu>",
-    "Sierra Moxon <smoxon@lbl.gov>",
-    "Bill Duncan <wdduncan@gmail.com>",
-    "Harshad Hegde <hhegde@lbl.gov>"
+  {name = "Chris Mungall", email = "cjmungall@lbl.gov"},
+  {name = "Harold Solbrig", email = "solbrig@jhu.edu"},
+  {name = "Sierra Moxon", email = "smoxon@lbl.gov"},
+  {name = "Bill Duncan", email = "wdduncan@gmail.com"},
+  {name = "Harshad Hegde", email = "hhegde@lbl.gov"},
 ]
-
-readme = "README.md"
-
+license = "CC0-1.0"
+license-files = ["LICENSE"]
 homepage = "https://github.com/linkml/linkml-runtime"
 repository = "https://github.com/linkml/linkml-runtime"
 documentation = "https://github.com/linkml/linkml-runtime"
-
+readme = "README.md"
 keywords = ["linkml", "metamodel", "schema visualization", "rdf", "owl", "yaml"]
-
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Environment :: Console",
     "Intended Audience :: Developers",
     "Intended Audience :: Science/Research",
     "Intended Audience :: Healthcare Industry",
-    "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
 ]
-
-packages = [
-    { include = "linkml_runtime" }
+include = ["linkml_runtime"]
+requires-python = ">=3.9"
```

Comment:
do we need to add an upper bound?

Reply:
Not for requires-python, see https://github.com/python-poetry/poetry/issues/10136